tasks/pre_install_checks.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Gather facts on installed packages
+  package_facts:
+
+- name: Fail when {{ default_firewall_manager }} is not installed
+  fail:
+    msg: '{{ ovirt_engine_setup_firewall_manager }} was chosen as default firewall manager but is not installed'
+  when: >
+    ovirt_engine_setup_firewall_manager == default_firewall_manager and
```

Comment:
Remove this line.

Reply:
Why remove? This is the whole point of this PR. If the use chose different firewall manager, then if don't care whether the default one is installed or not